staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -250,21 +322,74 @@ func (s *openAPIAggregator) loadApiServiceSpec(handler http.Handler, apiService
 		return nil
```

Comment:
> Also we are called her by the aggregator synchronously. If this download blocks, the main aggregation controller blocks. Do we want that? /cc @deads2k

We definitely do not.  This should be a separate controller, unreleated to the main api aggregation controller.